core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -2253,6 +2253,17 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey USER_FILE_READ_CACHE_TTL =
+      new Builder(Name.USER_FILE_READ_CACHE_TTL)
+          .setDefaultValue(Constants.NO_TTL)
+          .setDescription("Time to live for files loaded from UFS by a user, no ttl by default.")
+          .build();
+  public static final PropertyKey USER_FILE_READ_CACHE_TTL_EXPIRED_ACTION =
+      new Builder(Name.USER_FILE_READ_CACHE_TTL_EXPIRED_ACTION)
+          .setDefaultValue("DELETE")
+          .setDescription("When file's ttl is expired, the action performs on it."
```

Comment:
If I have a file "/file" only in the UFS, then Alluxio discovers it with TTL=1hour, TTL_ACTION=delete, does that mean that we'll delete the file from the UFS after 1 hour?

Reply:
Yes, i think so.

Reply:
I think write type THROUGH is the case as you described.